README.md

```diff
@@ -128,17 +148,222 @@ To start Parity Ethereum as a regular user using `systemd` init:
+```
+
+Replacing `<spec>` with one of the following (i.e. `cargo doc --package parity-ethereum --open`):
+
+* Parity Ethereum (EthCore) Client Application
+  ```bash
+  parity-ethereum
+  ```
+* Parity Ethereum Account Management, Hardware Wallet Support, Key Management Tool, and Keys Generator
+  ```bash
+  ethcore-accounts, ethkey-cli, ethstore, ethstore-cli, fake-hardware-wallet, hardware-wallets
+  ```
+* Parity Chain Specification
+  ```bash
+  chainspec
+  ```
+* Parity CLI Signer Tool & RPC Client
+  ```bash
+  cli-signer parity-rpc-client
+  ```
+* Parity Ethereum Ethash & ProgPoW Implementations
+  ```bash
+  ethash
+  ```
+* Parity (EthCore) Library
+  ```bash
+  ethcore
+  ```
+  * Parity Ethereum Blockchain Database, Generator, Configuration,
+Caching, Importing Blocks, and Block Information
+    ```bash
+    ethcore-blockchain
+    ```
+  * Parity Ethereum (EthCore) Contract Calls and Blockchain Service & Registry Information
+    ```bash
+    ethcore-call-contract
+    ```
+  * Parity Ethereum (EthCore) Database Access & Utilities, Database Cache Manager
+    ```bash
+    ethcore-db
+    ```
+  * Parity Ethereum Virtual Machine (EVM) Rust Implementation
+    ```bash
+    evm
+    ```
+  * Parity Ethereum (EthCore) Light Client Implementation
+    ```bash
+    ethcore-light
+    ```
+  * Parity Smart Contract based Node Filter, Manage Permissions of Network Connections
+    ```bash
+    node-filter
+    ```
+  * Parity Private Transactions
+    ```bash
+    ethcore-private-tx
+    ```
+  * Parity Ethereum (EthCore) Client & Network Service Creation & Registration with the I/O Subsystem
+    ```bash
+    ethcore-service
+    ```
+  * Parity Ethereum (EthCore) Blockchain Synchronization
+    ```bash
+    ethcore-sync
+    ```
+  * Parity Ethereum Common Types
+    ```bash
+    common-types
+    ```
+  * Parity Ethereum Virtual Machines (VM) Support Library
+    ```bash
+    vm
+    ```
+  * Parity Ethereum WASM Interpreter
+    ```bash
+    wasm
+    ```
+  * Parity Ethereum WASM Test Run
+    ```bash
+    pwasm-run-test
+    ```
+  * Parity EVM Implementation
+    ```bash
+    evmbin
+    ```
+  * Parity Ethereum IPFS-compatible API
+    ```bash
+    parity-ipfs-api
+    ```
+  * Parity Ethereum JSON Deserialization
+    ```bash
+    ethjson
+    ```
+  * Parity Ethereum State Machine Generalization for Consensus Engines
+    ```bash
+    parity-machine
+    ```
+* Parity Ethereum (EthCore) Miner Interface
+  ```bash
+  ethcore-miner parity-local-store price-info ethcore-stratum using_queue
+  ```
+* Parity Ethereum (EthCore) Logger Implementation
+  ```bash
+  ethcore-logger
+  ```
+* Parity Ethereum Client C-Bindings Library
+  ```bash
+  parity-clib
+  ```
+* Parity Ethereum JSON-RPC Servers
+  ```bash
+  parity-rpc
+  ```
+* Parity Ethereum (EthCore) Secret Store
+  ```bash
+  ethcore-secretstore
+  ```
+* Parity Updater Service
+  ```bash
+  parity-updater parity-hash-fetch
+  ```
+* Parity Utilities
```

Comment:
I don't think this is a fair label for what these are. These are not really "utilities": they are core pieces of functionality while "utility" sort of implies a "nice to have" piece of code. Maybe "Parity core libs"?

Reply:
I've addressed this in commit e87b8ba69cab57c2920a755f1ccac3e63423d7c8